test/FSharpTestProjects/TestApp/project.json

```diff
@@ -10,11 +10,11 @@
     "dependencies": {
         "TestLibrary": { "target": "project" },
         "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-151221",
-        "System.IO": "4.0.11-beta-23428",
-        "System.Console": "4.0.0-beta-23428",
-        "System.Runtime": "4.0.21-beta-23428",
-        "System.Diagnostics.Process": "4.1.0-beta-23428",
-        "Microsoft.NETCore.Runtime": "1.0.1-beta-23428"
+        "System.IO": "4.0.11-rc2-*",
```

Comment:
@davidfowl i updated references to latest HEAD of `test\TestProjects`, see previous comment
thx for review :smile: